code/controllers/subsystem/mapping.dm

```diff
@@ -386,7 +386,8 @@ GLOBAL_LIST_EMPTY(the_station_areas)
 			space_ruins_templates[R.name] = R
 
 /datum/controller/subsystem/mapping/proc/preloadShuttleTemplates()
-	var/list/unbuyable = generateMapList("[global.config.directory]/unbuyableshuttles.txt")
+	var/list/unbuyable = generateMapList("[global.config.directory]/shuttles_unbuyable.txt")
```

Comment:
This needs to be reflected in the repo. The old config file should be renamed accordingly.

Reply:
Clarify? I'm not sure what this means, my apologies. Do you mean update the CL?

Reply:
You renamed the file, but not the filepath in code to the file

Reply:
Wait, if I am not mistaken the file names are the same

Reply:
Yes, when I do a search in vscode I cannot find any references to unbuyableshuttles.txt, and the old file doesn't exist. 

Maybe the history appears weird from basing this off of someone elses PR i dunno.

Reply:
https://github.com/BeeStation/BeeStation-Hornet/blob/master/config/unbuyableshuttles.txt
https://github.com/BeeStation/BeeStation-Hornet/blob/master/config/unbuyableshuttles.txt

Are the files you need to rename.

Reply:
Those files do not exist in this branch

Reply:
Pull the latest master, it should resolve the issue I'm seeing.